package.json

```diff
@@ -35,6 +35,7 @@
     "babel-eslint": "^4.1.6",
     "babel-loader": "^6.2.1",
     "babel-plugin-transform-es2015-modules-umd": "^6.3.13",
+    "babel-plugin-transform-react-remove-prop-types": "^0.4.19",
```

Comment:
I know this is an old PR but should this module be installed? I don't see it referenced elsewhere...